src/js/player.js

```diff
@@ -2777,18 +2782,17 @@ class Player extends Component {
    * @listens keydown
    */
   handleKeyPress(event) {
+    if (!this.options_.userActions ||
+        this.options_.userActions.hotkeys === false) {
```

Comment:
I think the logic of this test is the wrong way round; `this.handleHotkeys(event)` should be called if `this.options_.userActions.hotkeys !== false`, right?